index.html.tmpl

```diff
@@ -246,10 +246,11 @@
                           Create a new branch from the staged-recipes master branch.
                       </li>
                       <li><i class="mega-octicon fa-3x octicon-pencil"></i>
-                          Add a new conda recipe in a new "recipes/[your-package-name]" directory.
-                          There is an example of a well-written recipe in "recipes/example" you can copy.
-                          <a href="docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml">Further guidance on writing good recipes</a>.
-                          <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">You can also test the recipe locally.</a>
+                          Add a new conda recipe in a new "recipes/[your-package-name]" directory. Please make sure to follow these
+                          <a href="https://conda-forge.org/docs/orga/guidelines.html">guidelines</a>.
+                          You can also refer to an example of a well-written recipe in "recipes/example".
+                          Further guidance on writing good recipes can be found <a href="docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml">here</a>.
+                          You can also <a href="docs/maintainer/adding_pkgs.html#running-tests-locally-for-staged-recipes">test your recipe locally.</a>
```

Comment:
The two instances of "you can also" are a little repetitive here. We could change the second one to "Test your recipe locally using [these instructions](LINK)."